native/dll/DShowCapture-2/ScanChannel.cpp

```diff
@@ -139,7 +139,7 @@ JNIEXPORT jstring JNICALL Java_sage_DShowCaptureDevice_scanBDAChannel0
 			   
 	char* scanResult = scanChannel( (CHANNEL_DATA*)pCapInfo->channel, deviceName, tuningString,  
 		                               countryRegion,  (int)streamType );
-	len = strlen( scanResult );
+	(size_t) len = strlen( scanResult );

```

Comment:
Not sure why you added the cast on the left hand side here...len should already be a size_t.

Reply:
Removed, got added before I did the change at Line 84